samples/suit/recovery/README.rst

```diff
@@ -0,0 +1,50 @@
+:orphan:
```

Comment:
This is orphaned, but still appearing in the docs because of the inclusion at the higher level: 
https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/suit.html

Reply:
Yes, this is/will be addressed in a different PR that is currently open, but thanks for catching!